app/build.gradle

```diff
@@ -14,17 +14,17 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 27
-        versionCode 140
-        versionName '4.1'
-        applicationId "troop.com.freedcam"
+        versionCode 139
+        versionName '4.2'
+        applicationId "debug.com.freedcam"

```

Comment:
its a bad idea to change this. with the appid and the signingkey android identify freedcam.

Reply:
I was debugging the app so that my daily used version doesn't conflict with debug version in my mobile. please ignore these changes. I've accidentally committed those files.